provers/sgx/prover/src/lib.rs

```diff
@@ -84,7 +84,11 @@ impl Prover for SgxProver {
             .get_or_init(|| async { cur_dir.join("secrets").join("priv.key") })
             .await;
         GRAMINE_MANIFEST_TEMPLATE
-            .get_or_init(|| async { cur_dir.join(CONFIG).join("sgx-guest.manifest.template") })
+            .get_or_init(|| async {
+                cur_dir
+                    .join(CONFIG)
+                    .join("sgx-guest.local.manifest.template")
```

Comment:
Oh, forget to update the file path. 👍
because in my side the setup is always be run outside the raiko.